create-cluster/action.yaml

```diff
@@ -58,6 +58,8 @@ runs:
           --set spec.kubeAPIServer.anonymousAuth=true \
           --set spec.networking.subnets[0].cidr=${{ inputs.node_cidr }} \
           --set spec.etcdClusters[0].manager.listenMetricsURLs=http://localhost:2382 \
+          --set spec.etcdClusters[*].etcdMembers[*].volumeSize=50 \
```

Comment:
Nit. The above setting specifies `spec.etcdClusters[0]`, while here `spec.etcdClusters[*]`. It could make sense to uniform them, unless there's a specific reason for the difference.

Reply:
I'm not sure if there are any measurements we're using that look at the metrics exported from etcd. There aren't many, but currently we're only exporting the metrics from the `main` etcd, and this change would include those from the `events` cluster as well.

Reply:
Ah, I thought there was only a single etcd cluster. Sounds good then.